pages/common/pgmtosbig.md

```diff
@@ -0,0 +1,8 @@
+# pgmtosbig
+
+> Convert a PGM image to the SBIG CCDOPS format.
+> More information: <https://netpbm.sourceforge.net/doc/pgmtosbig.html>.
+
+- Read a PGM file and produce a SBIG CCDOPS file as output:
```

Comment:
Changed it except [here](https://github.com/tldr-pages/tldr/pull/11404/files#diff-e962739532a6bec24688073e2056eccaa04172271f338eaffaa8ec2910537a1fR6) because of winicontopam's special behaviour (in that it only converts the best qulity file).